packages/lib/preprocess/src/extract-queries/extract-queries.cjs

```diff
@@ -73,10 +78,7 @@ const extractExternalQueries = (content, filename) => {
 		return true;
 	};
 
-	/**
-	 * @type Query[]
-	 */
-	return frontmatter.queries
+	return /** @type {string[]} */ (frontmatter.queries)
```

Comment:
There's a `.map` statement below with two branches, one for `string` and `object`, so I don't think this type is correct: We don't expect everything in this array to be strings.